test/aqua_test.jl

```diff
@@ -0,0 +1,4 @@
+@testitem "Aqua" begin
```

Comment:
This package doesn't use ReTestItems.jl, just Test.jl
```suggestion
@testset "Aqua" begin
```